scripts/buildtools/versions

```diff
@@ -4,4 +4,4 @@ github.com/algorand/msgp v1.1.52
 github.com/algorand/oapi-codegen v1.3.7
 github.com/go-swagger/go-swagger v0.25.0
 gotest.tools/gotestsum v1.6.4
-github.com/golangci/golangci-lint/cmd/golangci-lint v1.47.3
+github.com/golangci/golangci-lint/cmd/golangci-lint v1.49.0
```

Comment:
I was holding back this version until we upgrade to Go 1.19.. the formatter changed a lot

Reply:
i had that version locally, it requires go 1.19, so this didn't work out. (Your input is always welcome, just keep in mind that this is a Draft PR, it's not ready for review)